app/src/main/res/values/strings.xml

```diff
@@ -93,4 +94,57 @@
     <string name="firstrun_description">Automatically block online trackers while you browse. Then tap to erase visited pages, searches, cookies and passwords from your device.</string>
     <string name="firstrun_button">Ok, got it!</string>
     <string name="firstrun_footer">To be clear, sites and service you use can still know where you\'ve been.</string>
+
+    <!-- The page html title (i.e. the <title> tag content) -->
+    <string name="errorpage_title">Problem loading page</string>
+    <string name="errorpage_refresh">Try Again</string>
+
+    <string name="error_connectionfailure_title">Unable to connect</string>
+    <string name="error_connectionfailure_message">
+    <![CDATA[
+      <ul>
+        <li>The site could be temporarily unavailable or too busy. Try again in a few moments.</li>
+        <li>If you are unable to load any pages, check your mobile device’s data or Wi-Fi connection.</li>
+      </ul>
+    ]]>
+    </string>
+
+    <string name="error_connect_title">@string/error_connectionfailure_title</string>
+    <string name="error_connect_message">@string/error_connectionfailure_message</string>
+
+    <string name="error_timeout_title">The connection timed out</string>
+    <string name="error_timeout_message">@string/error_connectionfailure_message</string>
+
+    <string name="error_hostLookup_title">Server not found</string>
+    <string name="error_hostLookup_message"><![CDATA[
+      <ul>
+        <li>Check the address for typing errors such as
+          <strong>ww</strong>.example.com instead of
+          <strong>www</strong>.example.com</li>
+        <li>If you are unable to load any pages, check your device’s data or Wi-Fi connection.</li>
+      </ul>
+    ]]></string>
+
+
+    <string name="error_malformedURI_title">The address isn’t valid</string>
+    <string name="error_malformedURI_message">.... web addresses....</string>
```

Comment:
This string looks weird. Is this the correct one? If yes then this definitely needs a comment for translators. :)